src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -50,7 +50,7 @@ export class DictionaryConcepts extends Component {
     this.state = {
       conceptsCount: this.props.totalConceptCount,
       searchInput: '',
-      conceptLimit: 10,
+      conceptLimit: 20,
```

Comment:
Why are you increasing the concept limit?

Reply:
@dondrzzy Pagination limit should be 20 items according to the mockup. That is why I changed that limit.

Reply:
But it was not part of the scope. Why not create a separate ticket for this.